packages/server/lib/errors.coffee

```diff
@@ -881,6 +880,21 @@ getMsgByType = (type, arg1 = {}, arg2) ->
       """
       Failed to connect to Chrome, retrying in 1 second (attempt #{chalk.yellow(arg1)}/32)
       """
+    when "COULD_NOT_PARSE_ARGUMENTS"
+      if arg2
+        """
+        Cypress could not parse its arguments
```

Comment:
I feel like this error could use more information to help the user fix the problem, maybe something like this:

---

Cypress encountered an error while parsing the argument `argname`.

You passed: `value`

The error was: `error`

Reply:
these are good suggestions